Lib/fontTools/cffLib/CFFToCFF2.py

```diff
@@ -103,6 +105,9 @@ def _convertCFFToCFF2(cff, otFont):
         if program and program[-1] == "endchar":
             program.pop()
 
+    if removeUnusedSubrs:
+        cff.remove_unused_subroutines()
```

Comment:
Codecov suggests that existing tests do not cover this.

Reply:
Yeah I have to dig which glyph of NotoSansCJK or another font I was testing triggers this and subset it to a test.

Reply:
Adding that here: https://github.com/fonttools/fonttools/pull/3525